jwst/stpipe/step.py

```diff
@@ -935,7 +935,7 @@ def make_input_path(self, file_path):
         full_path = file_path
         if isinstance(file_path, str):
             original_path, file_name = split(file_path)
-            if not len(original_path):
+            if not len(original_path) and self.input_dir is not None:
```

Comment:
Inadvertently, I believe #2157 handles this. I would recommend waiting for that pr to merge, or merge it explicitly here, and ensure that it does the job.